lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -27,6 +27,7 @@
         HtmlUtils
     )
     {
+        var vent = _.extend({}, Backbone.Events);
```

Comment:
This is the 'middle-man' I was talking about in standup. Used on line 88 and 109 for listening and triggering respectively.